README.md

```diff
@@ -69,6 +80,15 @@ struct DemoAppApp: App {
                                                ignoreErrors: [],
                                                customDomainUrl: "",
                                                labels: ["String" : Any],
+                                               sampleRate: 100,
+                                               mobileVitalsFPSSamplingRate: 300, // minimum every 5 minute
+                                               instrumentations: [.navigation: true,
```

Comment:
also verify that he doesnt need to pass empty array.
he cannot pass at all, and it should be true